controls/cis_sle12.yml

```diff
@@ -1663,6 +1663,7 @@ controls:
     rules:
       - sshd_approved_macs=cis_sle12
       - sshd_use_approved_macs
+      - sshd_strong_macs=cis_sle12
```

Comment:
The rule `sshd_use_strong_macs` is used also in multiple other profiles in other products. You should explicitly assign the value of the new `sshd_strong_macs` variable in all the controls and profile files where the rule  `sshd_use_strong_macs` is selected.

Reply:
Did relevant changes in 40cf036f4dcc4877489f4bca239d443a6ad1d3cc but I guess that will need extra review from @ComplianceAsCode/oracle-maintainers,    @ComplianceAsCode/red-hatters and @ComplianceAsCode/ubuntu-maintainers if I made the right assumptions